### PR TITLE
clean up Juju requirements (and requirements in general)

### PR DESCRIPTION
**Pull request by ru-fu**
Signed-off-by: Ruth Fuchss <ruth.fuchss@canonical.com>

### CODE REVIEW

**Review by morphis — APPROVED** [review]
LGTM

File: requirements.md [hunk 1]
```diff
@@ -72,13 +70,27 @@ For external access to the Anbox Cloud Appliance, a couple of network ports must
...
+<a name="juju-version"></a>
+### Juju version
+
+Anbox Cloud requires [Juju](https://juju.is/) to be installed to manage the different components and their dependencies. The latest supported version is **2.9**.
+
+[note type="information" status="Note"]If you are using Anbox Cloud 1.11 or earlier, the required Juju version is **2.8**.[/note]
+
+You can install Juju 2.9 with the following command:
+
+    snap install --channel=2.9/stable juju
```
Comment (aguadoenzo) [inline requirements.md:87]:
```suggestion
    snap install --classic --channel=2.9/stable juju
```

**Review by aguadoenzo — CHANGES_REQUESTED** [review]

File: requirements.md (hunk 1 above)
Comment (ru-fu) [reply requirements.md:87]: Thanks - is this only for the install or also for refresh?

Comment (aguadoenzo) [reply requirements.md:87]: It's only for installation. Refreshing doesn't need user acknowledgement for the classic mode, as it was already granted on installation

**Pull request merged**